css/css-text/overflow-wrap/overflow-wrap-min-content-size-001.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Text Test: overflow-wrap: break-word and intrinsic sizing</title>
+<link rel="author" title="Florian Rivoal" href="http://florian.rivoal.net/">
+<link rel="help" href="https://drafts.csswg.org/css-text-3/#overflow-wrap-property">
+<meta name="flags" content="">
```

Comment:
What does it mean this empty flags tag ?

Reply:
it means that none of the possible flags listed in https://web-platform-tests.org/writing-tests/css-metadata.html apply.

Not including the flags meta at all might mean that too, or it might mean that the author forgot to consider the flags. So I prefer to include empty flags rather than leaving them out.